doc/build-instructions/Build_Instructions_V11.md

```diff
@@ -789,3 +790,344 @@ JCL      - d247952 based on jdk-11.0.6+6)
+for GNU/Linux 4.15.0, BuildID[sha1]=715cfde5a3bdc6ff31b6cc2e449f06df1b3c465a, not stripped
+```
+
+### 6. Cross-build the JDK
+:rocket:
+Now you're ready to cross-build OpenJDK with OpenJ9:
+```
+make all
+```
+
+A binary for the full developer kit (JDK without DDR support) is built and stored in the following directory:
+
+- **build/linux-riscv64-normal-server-release/images/jdk**
+
+Copy the JDK to a specified directory on the mounted Fedora OS image
+e.g.
+```
+cp -r jdk /<path_to>/<your_fedora_mount_directory>/home/<your_user_name>/new_dir
+```
+:bulb:
+Instead of copying the JDK into the Fedora OS image, you can also zip the JDK & upload it onto Fedora via the `scp`  command as follows:
+```
+[root@stage4 home/<your_user_name>/new_dir]# scp <user>@<the_IP_address_on_your_host_system>:/<path_to>/jdk.zip .
+[root@stage4 home/<your_user_name>/new_dir]# unzip -qq jdk.zip
+[root@stage4 home/<your_user_name>/new_dir]# jdk/bin/java -version
+```
+
+### 7. Test the JDK on Fedora/QEMU
+:rocket:
+Unmount the Fedora OS image after the cross-compilation is complete.
+```
+sudo umount /<path_to_your_fedora_mount_directory>
+```
+
+Boot Fedora via QEMU with the command `qemu-system-riscv64` at [step 2](#2-install-software-packages-for-the-cross-compilation) and change to your jdk directory on Fedora:
+```
+stage4 login: root
+Password: riscv
+Last login: Fri Oct  4 21:21:28 from 10.0.2.2
+[root@stage4 ~]# cd /home/<user_name>/new_dir/jdk
+```
+
+For a simple test, try running the `java -version` command.
+Run:
+```
+./bin/java -version
+```
+
+Here is some sample output:
+```
+$ ./bin/java -version
+openjdk version "11.0.5-internal" 2019-11-15
+OpenJDK Runtime Environment (build 11.0.5-internal+0-adhoc.root.openj9-openjdk-jdk11)
+Eclipse OpenJ9 VM (build master-41621b6b6, JRE 11 Linux riscv64-64-Bit Compressed References 20191119_000000 (JIT disabled, AOT disabled)
+OpenJ9   - 41621b6b6
+OMR      - 92c14ce2
+JCL      - e5937725d3 based on jdk-11.0.5+10)
+```
+
+:bulb:
+The JIT compiler for Riscv64 is not supported at the time of writing, in which case the JDK ends up with the same output by default, whether or not the `-Xint` option is specified on the command line.
+
+### 8. Build the JDK on Fedora/QEMU (Optionally if you want to enable the DDR support in the jdk)
+:rocket:
+The following building steps are only used with the DDR support enabled in compilation on Fedora.
+
+[1] Follow the steps at **4** to get all source required for building the JDK.
+e.g.
+```
+[root@stage4 <build_jdk_dir>] git clone https://github.com/ibmruntimes/openj9-openjdk-jdk11.git
+[root@stage4 <build_jdk_dir>] cd openj9-openjdk-jdk11
+[root@stage4 <build_jdk_dir>] bash get_source.sh
+```
+
+[2] Run the following configure script to set up the building environment on Fedora.
+```
+export JAVA_HOME=/<path_to>/<the_cross_built_jdk>
+export PATH="$JAVA_HOME/bin:$PATH"
+
+The boot JDK set up in `JAVA_HOME` is the cross-built jdk which is uploaded to Fedora.
+
```

Comment:
This should be normal text (the two export commands should be in literal text by themselves, and the configure command should be separate literal text below this sentence).

Reply:
Fixed.